x-pack/plugins/security_solution/public/timelines/components/edit_data_provider/helpers.tsx

```diff
@@ -97,9 +117,27 @@ export const getExcludedFromSelection = (selectedOperator: EuiComboBoxOptionOpti
 
   switch (selection) {
     case i18n.IS_NOT: // fall through
+    case i18n.IS_NOT_ONE_OF:
     case i18n.DOES_NOT_EXIST:
       return true;
     default:
       return false;
   }
 };
+
+/** Ensure that a value passed to ControlledDefaultInput is not an array */
+export const sanatizeValue = (value: string | number | unknown[]): string => {
+  if (Array.isArray(value)) {
+    // fun fact: value should never be an array
+    return value.length ? `${value[0]}` : '';
+  }
+  return `${value}`;
+};
+
+/** Ensure if DataProvider is default that value is not template-like */
+export const isValueFieldInvalid = (type: DataProviderType, value: string | number) => {
```

Comment:
Consider removing this validation function, because it may block valid use cases, per the following example:

It's possible to create a file with a leading `{`, per the following command line output:

```sh
➜  kibana git:(142435-add-is-one-of-operstor) touch ~/tmp/{a_file_with_a_leading_brace
➜  kibana git:(142435-add-is-one-of-operstor) ✗ ls -al ~/tmp
total 24
drwxr-xr-x    5 user  group   160 Nov 15 11:56 .
drwxr-x---+ 299 user  group  9568 Nov 15 11:56 ..
-rw-r--r--    1 user  group     0 Nov 15 11:56 {a_file_with_a_leading_brace
➜
```

The `isValueFieldInvalid` function prevents the user from adding a `file.name` `is` operator for the value `{a_file_with_a_leading_brace`, per the screenshot below:

![validation_error](https://user-images.githubusercontent.com/4459398/202004212-c18092a2-90b5-4c02-9c32-bc473a28eed7.png)

_Above: an incorrect validation error_

Reply:
`isValueFieldInvalid` was a remnant of code passed. it has now been removed :D